### DISCUSSION TITLE
Medium links in Search redirects to asset file instead of Medium

### DISCUSSION THREAD

**Discussion by strivn** [alshedivat/al-folio#2464] [Q&A] [answered]
Hi, 
First of all, thank you for the repository! 

So I just tried to setup a github page, and I'm not sure whether this is a bug or there's something I configured wrong. In my case: 

- The search function returns duplicated Medium posts. So suppose I have Medium posts {A, B}, the search returns {A, B, A, B}.  
- The linked Medium post is considered to be an asset (`example.github.io/medium-year/medium-title`) rather than a redirect link to the actual Medium post.

Anything I should check / fix? Thanks!

**Comment by george-gca** [comment, marked as answer]
These are bugs. Would you mind opening an issue for these?

**Reply by strivn** [reply]
sure, thanks for the answer
